### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DonRichards**
Updates a few items on the README
* Adds a column to indicate what is exposed by default
* Corrects a URL indicated for Solr
* Adds consistency with make command references (where a docker-compose command was referenced and could be substituted for a make command).

### CODE REVIEW

**Review by rosiel — COMMENTED** [review]
Thanks!

Can you throw in a line about where/how/that you're able to change it? (or does that belong elsewhere?) [Edit: or is that sufficiently implied by "by default"? If you'd rather not draw new users to thing they'd be better off not touching, I understand]

**Comment by DonRichards** [comment]
@rosiel OK, done.

File: README.md [hunk 1]
```diff
@@ -148,7 +156,7 @@ The former will create a starter site modeled off of https://sandbox.islandora.c
 If you already have a Drupal site but don't know how to export it,
 log into your server, navigate to the Drupal root, and run the following commands:
 
-- `drush config:export`
+- `make config-export`
```
Comment (rosiel) [inline README.md:159]: is it at all possible to move a non-ISLE site to ISLE? If so, these instructions seem like the place to look.

File: README.md [hunk 2]
```diff
@@ -87,17 +87,25 @@ password in the file
...
+| Cantaloupe  | [https://islandora.traefik.me/cantaloupe](https://islandora.traefik.me/cantaloupe)             |         Yes         |
+| Matomo      | [https://islandora.traefik.me/matomo/](https://islandora.traefik.me/matomo/)                   |         Yes         |
+| Code Server | [https://islandora.traefik.me:8443/](https://islandora.traefik.me:8443/)                       |         No          |
+
+> **Exposed**: the act of allowing the containerized application's ports to be accessible to the host machine (or public). In most cases this makes the specified URL available for the browser.
+
+To change a service exposed value edit the *.env* file. The values will start with "EXPOSE_". Make changes then rebuild the docker-compose file and then run the up command (even if it's already running) using the following commands.
+```shell
+make -B docker-compose.yml
+make up
```
Comment (rosiel) [inline README.md:107]: I followed these instructions and went from a functioning site (`starter_dev`) to this:

Screen Shot 2022-11-07 at 7 16 48 PM

I have to say the green is pretty, I've never seen it on the Drupal install screen before.

Comment (rosiel) [reply README.md:107]: The same thing happened when i started with a `make local` site.

Comment (rosiel) [reply README.md:107]: From Don on the Tech Call: maybe permissions got out of whack.

File: README.md (hunk 1 above)
Comment (rosiel) [reply README.md:159]: forget it. separate issue.

File: README.md (hunk 2 above)
Comment (rosiel) [reply README.md:107]: That was caused by the ENVIRONMENT not being set properly in my .env file. 🤦 

This is good.

**Review by rosiel — APPROVED** [review]

**Pull request merged**